lib/processManager/processManager.js

```diff
@@ -599,7 +611,8 @@ processManager.start = function (cb) {
 
 processManager.reload = function (cb) {
 	if (workerManager) {
-		workerManager.recycle(cb);
+		// Recycle one worker at a time
```

Comment:
Why one at a time?

Reply:
Right now, we kill the processes after we have reloaded. But if we want to have a second process to have the same "id", we should probably kill first, then restart, which means we should probably not reload all at once.

However, if this is not desirable, we'll still need to decide what we want in terms of reload logic; right now, it is hard-coded to always reload 4 at a time, which I don't think is a constant that necessarily make sense.

Reply:
What should I do with this?